src/validators/deep-equal.js

```diff
@@ -0,0 +1,15 @@
+const { isDeepEqual } = require('../helpers');
+
+function deepEqual(value, { errorMessage = null } = {}) {
```

Comment:
rather than a file per validator, can we store them in relevant modules? all comparator based ones in `src/validators/equality.js` and exported in an object `module.exports = { equal, deepEqual, greaterThan, greaterThanOrEqual, ... }` etc.

Reply:
@bassjacob I am planning to do this in index file while exporting